app/assets/tailwind/application.css

```
@@ -1 +1,92 @@
+/* ========================================
+   Typography
+   ======================================== */
+
+body {
+  font-family:
+    "Hiragino Maru Gothic ProN",
+    "Yu Gothic",
+    "YuGothic",
+    sans-serif;
+
+  color: #26332E;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)